server/debug/src/process.rs

```diff
@@ -54,6 +55,10 @@ impl Child {
         self.write_memory(breakpoint.address, &breakpoint.saved)?;
         Ok(())
     }
+
+    pub fn new(handle: winapi::winnt::HANDLE) -> io::Result<Child> {
```

Comment:
This should be called `attach`, take a `RawHandle` rather than a `winapi::HANDLE`, and should be where all the setup happens, up to and including `DebugActiveProcess`. That way we can reimplement it for other platforms later without changing code outside of the `debug` crate.